### DISCUSSION TITLE
CDIPS file name changes - any impact to lightkurve support?

### DISCUSSION THREAD

**Discussion by scfleming** [lightkurve/lightkurve#1196] [Q&A] [answered]
To make the products more consistent with other TESS HLSP light curves, the CDIPS light curves at MAST will have a slight naming change: whereas before the Sector was indicated with just a 4-digit zero-pad in the file name, example:

-0014

it will now include an 's', example:
-s0014

This will change the file name itself and the Observation ID.  I do not _think_ it will impact lightkurve, but wanted to post here just in case.  Note we will also be going back and changing Sectors 1-13 with this update, but will get Sectors 14-26 releases first with this naming convention before re-doing Sectors 1-13.

Tagging @hmlewis-astro

**Comment by barentsen** [comment, marked as answer]
@scfleming Thank you for the heads up!

Good news: I do not expect any impact on Lightkurve.  I reviewed `lightkurve/io/cdips.py` and concluded that no information is parsed or extracted from CDIPS file names.

The data search features use the `sequence_number` field offered by the Portal API to search and display sector numbers.  For example, searching for a CDIPS light curve by sector number results in the following call to astroquery behind the scenes:
```python
astroquery.mast.Observations.query_criteria(..., sequence_number=sector, provenance_name="CDIPS")
```

**Reply by scfleming** [reply]
Awesome, thanks!  We might temporarily bring down Sectors 1-13 when we are ready to re-ingest with the new names.  Hopefully that will be done within a day or two at most (CDIPS is fairly small).

**Reply by orionlee** [reply]
A very minor issue is that if users rely on data products cached locally (default at `~/.lightkurve-cache/`), the cached data products would no longer valid due to the file name change. It probably won't impact anyone in practice, as any subsequent search + download would re-populate the cache with updated files.

**Reply by barentsen** [reply]
Oops, it turned out that the CDIPS changes did affect Lightkurve in the sense that we had some urls/filenames hard-coded as part of the unit tests.

Fortunately this was promptly fixed by @orionlee via #1203, and did not actually affect Lightkurve's functionality beyond the unit tests.
